packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -95,6 +95,35 @@ describe('cluster', () => {
       MasterUserPassword: 'tooshort',
       VpcSecurityGroupIds: [{ 'Fn::GetAtt': ['DatabaseSecurityGroup5C91FDCB', 'GroupId'] }],
     });
+
+    expect(cluster.instanceIdentifiers).toHaveLength(1);
+    expect(stack.resolve(cluster.instanceIdentifiers[0])).toEqual({
+      Ref: 'DatabaseInstance1844F58FD',
+    });
+
+    expect(cluster.instanceEndpoints).toHaveLength(1);
+    expect(stack.resolve(cluster.instanceEndpoints[0])).toEqual({
+      hostname: {
+        'Fn::GetAtt': ['DatabaseInstance1844F58FD', 'Endpoint.Address'],
+      },
+      port: {
+        'Fn::GetAtt': ['DatabaseB269D8BB', 'Endpoint.Port'],
+      },
+      socketAddress: {
+        'Fn::Join': [
+          '',
+          [
+            {
+              'Fn::GetAtt': ['DatabaseInstance1844F58FD', 'Endpoint.Address'],
+            },
+            ':',
+            {
+              'Fn::GetAtt': ['DatabaseB269D8BB', 'Endpoint.Port'],
+            },
+          ],
+        ],
```

Comment:
Let's maybe save some lines here:

```suggestion
        'Fn::Join': ['', [
          { 'Fn::GetAtt': ['DatabaseInstance1844F58FD', 'Endpoint.Address'] },
          ':',
          { 'Fn::GetAtt': ['DatabaseB269D8BB', 'Endpoint.Port'] },
        ]],
```